about.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About page</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <header>
+      <nav id="web">
+        <a href="index.html"><img id="logo" src= './images/logo.jpg' alt="logo"></a>
+        <nav id="navigator">
+          <a href="./about.html">About</a>
+          <a href="#program">Programs</a>
+          <a>Join</a>
+          <a href="#black">Sponser</a>
+          <a>News</a>
+        </nav>
+      </nav>
+    </header>
+
+    <main>
+      <section id="intro">
+        <img id="intro-background" src="./images/main-bg.png" alt="background">
+        <section id="headline">
+          <h2 id="welcome">Welcome to theater!</h2>
+          <h1>My beloved, who is she?<br>حبيبتي من تكون</h1>
+        </section>
+        <p id="bio">We always wondered who the Nightingale's lover is! Will we find the answer through this brilliant panoramic show, which revolves around the biography of Abdel Halim Hafez? We always wondered who the Nightingale's lover is! Will we find the answer through this brilliant panoramic show, which revolves around the biography of Abdel Halim Hafez? We always wondered who the Nightingale's lover is! Will we find the answer through this brilliant panoramic show, which revolves around the biography of Abdel Halim Hafez? We always wondered who the Nightingale's lover is! Will we find the answer through this brilliant panoramic show, which revolves around the biography of Abdel Halim Hafez?</p>
+        <h3 id="date">Tuesday 17/May - Wednesday 25/May •2022•</h3>
+      </section>
+
+      <section class="festiv">
+        <h2>Festival Logo</h2>
+        <hr>
+        <p>Time: from 06:30 PM to 09:15 PM<br>
+          City: Jeddah<br>
+          Venue: The World Theatre, City Walk</p>
+        <img id="jed-logo" src="./images/JS-AR-Logo.svg" alt="festival logo">
+      </section>
+      <hr>
+      <section class="prev">
+        <h2>Other Events</h2>
+        <hr>
+        <div class="event">
+          <div class="shade"><img src="./images/event1.png" alt="snapshot"></div>
+          <div class="shade"><img src="./images/event2.png" alt="snapshot"></div>
+        </div>
+      </section>
+    </main>
+    <script src="script.js"></script>
+    <footer id="black">
+      <img src="./images/JS-AR-Logo.svg" alt="festival logo">
+      <p>Time: from 06:30 PM to 09:15 PM<br><br>
+        City: Jeddah<br>
+        Venue: The World Theatre, City Walk</p>
+    </footer>
```

Comment:
- [ ] Kindly add your footer to the mobile version of your about page 👍🏿 
### OPTIONAL 

- [ ]  Currently there is no footer on the index page but you can add it, it would be nice to have a footer on all pages  🤗

Reply:
I chose the mobile version design on the left which does not have a footer. I understand your advice can enhance my project. I'm afraid if I followed it, the next reviewer will reject it.